src/transformers/quantizers/quantizer_torchao.py

```diff
@@ -166,7 +166,8 @@ def is_serializable(self):
 
     @property
     def is_trainable(self):
-        # torchao does not have official support for QAT (Quantization Aware Training)
```

Comment:
so we support both QAT and have experimental support for quantized training as well

Reply:
We should probably create another propriety like `is_qat_trainable` and change this to `is_peft_trainable` to distinguish these two types of training. 
Do you have a script for the QAT training ?

Reply:
Sorry for delay but here you go https://github.com/pytorch/ao/tree/main/torchao/quantization/prototype/qat also cc @andrewor14